index.html

```diff
@@ -151,6 +151,7 @@ <h2>Sponsors</h2>
         <div class="sponsors-wrapper">
             <div class="sponsor">
                 <span class="vertical-helper"></span><a href="https://developer.ibm.com/technologies/node-js/"><img class="small-img" src="./assets/images/sponsors/IBM-logo.svg" alt="IBM Logo"></a>
+                <span class="vertical-helper"></span><a href="https://risingstack.com"><img class="small-img" src="./assets/images/sponsors/RisingStack-logo.svg" alt="RisingStack Logo"></a>
```

Comment:
Please move this into a new <div class="sponsor">

Reply:
fixed